puppet/modules/profiles/manifests/base.pp

```diff
@@ -20,4 +24,41 @@
   class { 'foreman_proxy::plugin::remote_execution::ssh_user':
     manage_user => true,
   }
+
+  # Configure Grafana-based monitoring
```

Comment:
I wonder if it makes sense to keep this in a private class to keep the base profile size limited

Reply:
you mean something like `profile::base::monitoring` or a totally dedicated `monitoring_client` module that gets included here?

Reply:
I was thinking `profiles::base::monitoring` because I'm lazy and don't want to introduce another module. If it gets larger we can always still do that.

Reply:
like this?